sharejs-ace/client.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Created by dsichau on 27.04.16.
+ */
+
+import { Template } from 'meteor/templating'
+import { Blaze } from 'meteor/blaze'
+import { ShareJSConnector } from 'meteor/mizzao:sharejs'
+
+require('ace-builds/src/ace');
+ace.require('ace/config').set('basePath', 'https://cdn.jsdelivr.net/ace/1.2.2/min');
```

Comment:
Does anyone now of a better way to solve this. At the moment this includes an external cdn.